components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -1941,6 +1942,9 @@ def full_viewer(request, iid, conn=None, **kwargs):
     @return:            html page of image and metadata
     """
 
+    server_id = request.session['connector'].server_id
+    server_name = Server.get(server_id).server
```

Comment:
@manics FastCGI used manage.py to start workers, wsgi uses different application handler, where settings are not imported. if you remove https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroWeb/omeroweb/manage.py#L38 it will fail with exactly the same error.